src/firmware/app/control/wheel_controller.h

```diff
@@ -0,0 +1,93 @@
+#pragma once
+
+#include "firmware/shared/circular_buffer.h"
+
+// wheel_controller_difference_equation.h
```

Comment:
what's this comment?

Reply:
Ah, I think Clion auto generated that one. Removed 👍